### PR TITLE
🐇

### PR DESCRIPTION
**Pull request by mlsteele**
Some rabbits. Not all rabbits. After this merges please call out missing rabbits.

@keybase/react-hackers @keybase/picnicsquad

### CODE REVIEW

File: shared/team-building/service-tab-bar.desktop.tsx [hunk 1]
```diff
@@ -237,6 +244,9 @@ const styles = Styles.styleSheetCreate(() => ({
   tabBarContainer: {
     minHeight: 30,
   },
+  wonderland: {
+    color: Styles.globalColors.white,
```
Comment (buoyad) [inline shared/team-building/service-tab-bar.desktop.tsx]: ?

Comment (mlsteele) [reply shared/team-building/service-tab-bar.desktop.tsx]: The rabbit appears transparent with the default transparent text color. This makes it full opacity.

File: shared/chat/conversation/bottom-banner/index.tsx [hunk 2]
```diff
@@ -39,7 +40,11 @@ const InviteBanner = ({users, openSMS, openShareSheet, usernameToContactName}: I
     return (
       <BannerBox color={Styles.globalColors.blue} gap="xtiny">
         <BannerText>Last step: summon {theirName}!</BannerText>
-        <Button label="Send install link" onClick={() => openSMS(users[0].slice(0, -6))} mode="Secondary" />
+        <Button
+          label={Flags.wonderland ? '🐇 Send install link' : 'Send install link'}
```
Comment (xgess) [inline shared/chat/conversation/bottom-banner/index.tsx]: there are three of these buttons in this file. you've only got the one that's mobile and one-on-one. are we doing this for desktop also? assuming no, then probably just need one more. down a few lines.

Comment (mlsteele) [reply shared/chat/conversation/bottom-banner/index.tsx]: good catch. Afaik the 🐇 task includes 🐇 on desktop. Will get in future PR

**Pull request merged**